instrumentation/grpctrace/interceptor.go

```diff
@@ -86,7 +86,11 @@ func UnaryClientInterceptor(tracer trace.Tracer) grpc.UnaryClientInterceptor {
 			ctx, method,
 			trace.WithSpanKind(trace.SpanKindClient),
 			trace.WithAttributes(peerInfoFromTarget(cc.Target())...),
-			trace.WithAttributes(standard.RPCServiceKey.String(serviceFromFullMethod(method))),
+			trace.WithAttributes(
+				standard.RPCSystemGRPC,
+				standard.RPCServiceKey.String(method),
```

Comment:
My reading of https://github.com/open-telemetry/opentelemetry-specification/pull/604 is that if the full span name is `$package.$service/$method`, then the `RPCServiceKey` attribute should contain `$package.$service`, and `RPCMethodKey` should contain `$method`.

I admit that the document doesn't have a clear example, so may be open to interpretation.

Reply:
On the second spec read, I think you and @Aneurysm9 are right, `rpc.service` being `$package.$service` is what I would expect as well